front/lib/models.ts

```diff
@@ -419,6 +419,50 @@ DataSource.init(
 );
 User.hasMany(DataSource);
 
+export class Run extends Model<
+  InferAttributes<Run>,
+  InferCreationAttributes<Run>
+> {
+  declare id: CreationOptional<number>;
+  declare createdAt: CreationOptional<Date>;
+  declare updatedAt: CreationOptional<Date>;
+
+  declare dustRunId: string;
+  declare userId: ForeignKey<User["id"]>;
+  declare appId: ForeignKey<App["id"]>;
+}
+
+Run.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    createdAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+    updatedAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW,
+    },
+    dustRunId: {
```

Comment:
I think we want the runType as well? So that we can reproduce the UI of showing the list per type.